app/models/agents/pushover_agent.rb

```diff
@@ -12,42 +12,41 @@ class PushoverAgent < Agent
 
       **You need a Pushover API Token:** [https://pushover.net/apps/build](https://pushover.net/apps/build)
 
-      **You must provide** a `message` or `text` key that will contain the body of the notification. This can come from an event or be set as a default. Pushover API has a `512` Character Limit including `title`. `message` will be truncated.
-
       * `token`: your application's API token
       * `user`: the user or group key (not e-mail address).
       * `expected_receive_period_in_days`:  is maximum number of days that you would expect to pass between events being received by this agent.
 
-      Your event can provide any of the following optional parameters or you can provide defaults:
+      The following options are all Liquid templates which evaluated values will be posted to Pushover API.  Only the `message` parameter is required, and if it is blank API call is omitted.
```

Comment:
"The following options are all Liquid templates *whose* evaluated values will be posted to *the* Pushover API."